jobs/JGLOBAL_MARINE_BMAT

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+export DATA=${DATA:-${DATAROOT}/${RUN}marinebmat_${cyc}}
+# source config.base, config.ocnanal and config.marinebmat
+# and pass marinebmat to ${machine}.env
+source "${HOMEgfs}/ush/jjob_header.sh" -e "marinebmat" -c "base ocnanal marinebmat"
+
+# If ensemble perturbations are used, save them in a DATAenspert directory that will
+# exist throughout the DA cycle
+if (( 10#${NMEM_ENS:-0} > 0 )); then
+  export DATAjob="${DATAROOT}/${RUN}marinebmat.${PDY:-}${cyc}"
+  export DATAenspert="${DATAjob}/enspert"
+  if [[ ! -d "${DATAenspert}" ]]; then mkdir -p "${DATAenspert}"; fi
+fi
+
+##############################################
+# Set variables used in the script
+##############################################
+# shellcheck disable=SC2153
+GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
+export GDUMP="gdas"
+export GDUMP_ENS="enkf${GDUMP}"
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Generate COM variables from templates
+RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
+   COMIN_OCEAN_HISTORY_PREV:COM_OCEAN_HISTORY_TMPL \
+   COMIN_ICE_HISTORY_PREV:COM_ICE_HISTORY_TMPL
+
+RUN=${GDUMP_ENS} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
+    COMIN_OCEAN_HISTORY_ENS_PREV:COM_OCEAN_HISTORY_TMPL \
+    COMIN_ICE_HISTORY_ENS_PREV:COM_ICE_HISTORY_TMPL
+
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMOUT_OCEAN_BMATRIX:COM_OCEAN_BMATRIX_TMPL \
+   COMOUT_ICE_BMATRIX:COM_ICE_BMATRIX_TMPL
+
+mkdir -p "${COMOUT_OCEAN_BMATRIX}"
+mkdir -p "${COMOUT_ICE_BMATRIX}"
+
+###############################################################
+# Run relevant script
+
+EXSCRIPT=${GDASMARINEBMATRUNPY:-${SCRgfs}/exglobal_marine_bmat_run.py}
```

Comment:
why `_run`?
EE2 will make us change this to match the j-job name.

Reply:
ok, will push the changes after I'm done testing